extensions/SemanticMediaWiki/load.php

```diff
@@ -77,11 +77,10 @@ public static function initExtension() {
 		// Wikia change - begin
 		// SUS-1232: allow SemanticMediaWiki config variables to be customized via WikiFactory
 
-		/* @var $smwDefaults Array */
-
-		foreach( $smwDefaults as $variableName => $variableDefaultValue ) {
+		/** @var array $smwDefaults */
+		foreach ( $smwDefaults as $variableName => $variableDefaultValue ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$smwDefaults" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)